SchedulerBot.Client/Parsers/EventParser.cs

```diff
@@ -205,8 +205,9 @@ private static void ParseEventInputFlags(string[] args, ref Event evt, string ti
             }
         }
 
-        private static void ParseEventTimestamps(string bodyString, string timezone, ref Event evt)
+        private static void ParseEventTimestamps(string bodyString, string timezone, ref Event evt, bool update = false)
```

Comment:
Can you change `update` to `isUpdate`? That makes it more obvious that it's a boolean.